src/lib.rs

```diff
@@ -77,6 +107,7 @@ impl CorsMiddleware {
     pub fn with_allow_any() -> Self {
```

Comment:
And also here.

    #[deprecated(since="0.7.0", note="please use the `CorsMiddlewareBuilder` instead")]